components/pages/PlantInfo.js

```diff
@@ -16,28 +16,27 @@ export default class PlantInfo extends Component {
   state = {
     plantInfo: {}
    }
-  
+
   componentDidMount() {
     this.getPlantId(this.props.route.params.id)
   }
 
   getPlantId = (id) => {
     api.getPlantById(id)
-    .then(response => this.setState({plantInfo: response.data}))
-    .catch(err => console.log(err))
+      .then(response => this.setState({plantInfo: response.data}))
+      .catch(err => console.log(err))
```

Comment:
This shouldn't need a tab because technically this is all one long chain of events. The .then and .catch are not nested children of the api call.

Reply:
I think it's more readable with the indentation.

Reply:
I accept that